CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # Changelog
 
+## 2021.10.110 (11 November 2021)
+
```

Comment:
Do you want to take in Jim's changes for listing out versions in the changelog?

Reply:
Lets do that in the next official relase